2025-08-18 10:11:20 [INFO    ] src.utils.excel: Logger configurado. Arquivo de log: logs/catalog_20250818_101120.log
2025-08-18 10:11:20 [INFO    ] src.utils.images: Logger configurado. Arquivo de log: logs/catalog_20250818_101120.log
2025-08-18 10:11:20 [INFO    ] src.catalog_builder: Logger configurado. Arquivo de log: logs/catalog_20250818_101120.log
2025-08-18 10:11:20 [INFO    ] __main__: Logger configurado. Arquivo de log: logs/catalog_20250818_101120.log
2025-08-18 10:11:20 [INFO    ] __main__: Iniciando geração do catálogo com layout manual
2025-08-18 10:11:20 [INFO    ] src.catalog_builder: Iniciando geração do catálogo...
2025-08-18 10:11:20 [DEBUG   ] src.catalog_builder: Argumentos: {'excel': 'data/produtos.xlsx - Planilha1.csv', 'img_excel': 'data/base_imagens.xlsx', 'imagens': 'data/imagens', 'template': 'template/template_catalogo.pptx', 'out': 'catalogo_gerado.pptx', 'layout': 'manual', 'cols': None, 'rows': None, 'skip_download': False, 'include_no_image': False, 'max_products': 0}
2025-08-18 10:11:20 [INFO    ] src.utils.excel: Carregando planilha: data/produtos.xlsx - Planilha1.csv
2025-08-18 10:11:20 [ERROR   ] src.utils.excel: Arquivo não encontrado: data/produtos.xlsx - Planilha1.csv
2025-08-18 10:11:20 [ERROR   ] __main__: Erro fatal ao gerar catálogo: [Errno 2] No such file or directory: 'data/produtos.xlsx - Planilha1.csv'
Traceback (most recent call last):
  File "C:\Users\gustavo\Documents\Automation project\Catalogue-Automation\src\main.py", line 73, in main
    build_catalog(args, layout_config)
  File "C:\Users\gustavo\Documents\Automation project\Catalogue-Automation\src\catalog_builder.py", line 93, in build_catalog
    df_prod, c_sku, c_nome, c_cat, c_marca = load_product_data(args.excel)
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gustavo\Documents\Automation project\Catalogue-Automation\src\utils\excel.py", line 22, in load_product_data
    df = pd.read_excel(file_path)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gustavo\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\io\excel\_base.py", line 495, in read_excel
    io = ExcelFile(
         ^^^^^^^^^^
  File "C:\Users\gustavo\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\io\excel\_base.py", line 1550, in __init__
    ext = inspect_excel_format(
          ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gustavo\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\io\excel\_base.py", line 1402, in inspect_excel_format
    with get_handle(
         ^^^^^^^^^^^
  File "C:\Users\gustavo\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\io\common.py", line 882, in get_handle
    handle = open(handle, ioargs.mode)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'data/produtos.xlsx - Planilha1.csv'
